demisto_sdk/commands/common/timers.py

```diff
@@ -0,0 +1,119 @@
+# STD python packages
+import logging
+import time
+from collections import namedtuple
+from functools import wraps
+from pathlib import Path
+
+# Third party packages
+from tabulate import tabulate
+
+# Local packages
+from demisto_sdk.commands.common.logger import Colors
+
+# from typing import Dict, List
+
+logger = logging.getLogger('demisto-sdk')
+
+CSV_HEADERS = ['Function', 'Avg', 'Total', 'Call count']
+
+StatInfo = namedtuple("StatInfo", ["total_time", "call_count", "avg_time"])
+
+registered_timers: dict = {}
+
+
+def timer(group_name='Common'):
+    """
+    Decorate the functions using this decorator for time measurement
+
+    Arg:
+        group_name(str): the timers group name this timer will add to (individual group for each command)
+    """
+    def group_timer(func):
+        total_time = 0.0
+        call_count = 0
+
+        @wraps(func)
+        def wrapper_timer(*args, **kwargs):
+            nonlocal total_time, call_count
+
+            tic = time.perf_counter()
+            value = func(*args, **kwargs)
+            toc = time.perf_counter()
+
+            elapsed_time = toc - tic
+
+            total_time += elapsed_time
+            call_count += 1
+
+            return value
+
+        def stat_info():
+            return StatInfo(total_time, call_count, total_time / call_count if call_count != 0 else 0)
+
+        wrapper_timer.stat_info = stat_info  # type: ignore[attr-defined]
+
+        if group_name not in registered_timers:
+            registered_timers[group_name] = set()
+
+        registered_timers[group_name].add(wrapper_timer)
```

Comment:
```suggestion
        registered_timers.setdefault(group_name, set()).add(wrapper_timer)
```

or define `registered_timers` as `defaultdict(set)` and here it can be downsized to:
```suggestion
        registered_timers[group_name].add(wrapper_timer)
```